### DISCUSSION TITLE
tutorials/intro-dgs/02-graphql-basics

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#100] [course feedback]
# tutorials/intro-dgs/02-graphql-basics

How GraphQL works, and the components that make up a GraphQL server

https://www.apollographql.com/tutorials/intro-dgs/02-graphql-basics

**Comment by melwindmello** [comment]
Small typo - repetitive "the" in statement "we'll take some time to zoom into the the document"

**Reply by lizhennessy** [reply]
Thanks for pointing this out, @melwindmello ! We're on it. 🙌

**Comment by punjprakash** [comment]
Great work here ! loved the concise and simple explanation!
